CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 - [10725](https://github.com/vegaprotocol/vega/issues/10725) - Batch proposal votes to contain `ELS` per market.
 - [10744](https://github.com/vegaprotocol/vega/issues/10744) - Prevent governance suspension of a market already governance suspended. 
 - [10374](https://github.com/vegaprotocol/vega/issues/10374) - Ledger entries did not return data when filtering by transfer id.
+- [10635](https://github.com/vegaprotocol/vega/issues/10635) - Add support for tick size
```

Comment:
Given this is a breaking change in a market-proposal can you put it in the breaking-change section and tag the PR/ticket as `breaking-change` please?